Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu
+
+# set up timezone
+RUN echo "America/New_York" > /etc/timezone
+
+# install required packages
+RUN apt-get update \
+&& DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    python3 \
+    python3-dev \
+    python3-pip \
+    ffmpeg python3-tk \
```

Comment:
I think adding `--no-install-recommends` at the end of _apt-get install_ should skip all the unnecessary installations